<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniAudio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../myimage.qrc">
    <normaloff>:/images/images/MiniAudio.png</normaloff>:/images/images/MiniAudio.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="7" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="5" column="2" colspan="3">
     <widget class="QLabel" name="MusicName">
      <property name="text">
       <string>         </string>
      </property>
      <property name="buddy">
       <cstring></cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2" rowspan="2" colspan="3">
     <widget class="CurvePlot" name="ploterAll" native="true"/>
    </item>
    <item row="0" column="2" rowspan="3" colspan="3">
     <widget class="CurvePlot" name="ploter" native="true">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLabel" name="PositionLable">
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1293</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionAdd"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionNext"/>
    <addaction name="actionLast"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>效果</string>
    </property>
    <addaction name="actionTone"/>
    <addaction name="actionFrequency"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutThis"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionLast"/>
   <addaction name="actionPlay"/>
   <addaction name="actionNext"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionTone"/>
   <addaction name="actionFrequency"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/plus.png</normaloff>:/images/images/plus.png</iconset>
   </property>
   <property name="text">
    <string>添加文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionTone">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/tone.png</normaloff>:/images/images/tone.png</iconset>
   </property>
   <property name="text">
    <string>音调</string>
   </property>
  </action>
  <action name="actionFrequency">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/freq.png</normaloff>:/images/images/freq.png</iconset>
   </property>
   <property name="text">
    <string>频率</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/qt.png</normaloff>:/images/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/play.png</normaloff>:/images/images/play.png</iconset>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/pause.png</normaloff>:/images/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/next.png</normaloff>:/images/images/next.png</iconset>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
  </action>
  <action name="actionLast">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/last.png</normaloff>:/images/images/last.png</iconset>
   </property>
   <property name="text">
    <string>上一首</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/stop.png</normaloff>:/images/images/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actionAboutThis">
   <property name="icon">
    <iconset resource="../../myimage.qrc">
     <normaloff>:/images/images/MiniAudio.png</normaloff>:/images/images/MiniAudio.png</iconset>
   </property>
   <property name="text">
    <string>关于本产品</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CurvePlot</class>
   <extends>QWidget</extends>
   <header location="global">CurvePlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../myimage.qrc"/>
 </resources>
 <connections/>
</ui>
